### DISCUSSION TITLE
issue with npm install: npm ERR! code E401

### DISCUSSION THREAD

**Discussion by manorlh** [vercel/community#7429] [Help]
### Summary

there is issue with npm install ony deployment 
the error in the logs is : npm ERR! code E401
npm ERR! Incorrect or missing password.
npm ERR! If you were trying to login, change your password, create an
npm ERR! authentication token or enable two-factor authentication then
npm ERR! that means you likely typed your password in incorrectly.
npm ERR! Please try again, or recover your password at:
npm ERR!     https://www.npmjs.com/forgot
npm ERR! 

### Steps to Reproduce

deploy my react app

**Comment by pawlean** [comment]
Hi @manorlh!

From the error message, it looks like you have incorrect credentials. It's worth checking this and trying again.

**Reply by manorlh** [reply]
Yes, but I have never configured credentials in Vercel. I thought it was available out of the box. Where can I configure credentials in Vercel?

thanks

**Reply by pawlean** [reply]
I'm sorry I don't understand. The error you're seeing above is an issue with authenticating with npmjs.com.

**Reply by manorlh** [reply]
yep but it happens in Vercel deployment 
<img width="1262" alt="Screenshot 2024-07-11 at 16 38 09" src="https://github.com/vercel/community/assets/44364426/d5a30b1d-0a09-47c6-bbde-40ffe2d41893">

**Comment by sergio-cravas** [comment]
Any update on this? I'm stuck with the same problem

**Comment by sukhiboparai** [comment]
I am also struck with the issue. Anything resolved yet?

**Comment by amyegan** [comment]
Does your project have a `.npmrc` file pointing to a different registry? That sometimes happens if you're using a computer at a company that requires use of it's own private package registry and could cause a login error like this.

[a comment by an automated account is not shown]
